### ISSUE TITLE
embedding_size arg should be of type Int

### ISSUE THREAD

**Issue by theweaklink** [davidsandberg/facenet#270]
Hi there,

I ran into a minor problem that would be easy to fix. In `src/train_tripletloss.py`:
```python
parser.add_argument('--embedding_size', type=float, help='Dimensionality of the embedding.', default=128)
```

`type=float` should simply be replaced with `type=int` otherwise tensorflow will complain about it:
```python
raise ValueError('num_outputs should be int or long, got %s.', num_outputs)
```

hope it helps.
cheers

**Comment by davidsandberg** [comment]
You're right! Will fix this in next update. Thanks!!

**Comment by davidsandberg** [comment]
Fixed in cb5c750d890949dabd990289e182e1c72c7ed17f.
